axlearn/cloud/gcp/pathways_utils.py

```diff
@@ -183,7 +183,7 @@ def _build_pathways_head_container(self) -> dict:
         head_container["env"] = env_list
 
         cpu_req = f"{float(self.config.pathways_head_cpu) * 1000}m"
-        mem_req = f"{self.config.pathways_head_mem}Gi"
+        mem_req = f"{int(self.config.pathways_head_mem)}Gi"
```

Comment:
Thanks. Should we use float, so it supports more fine-grained granularity.

We should probably just change str to float in the config

```
        pathways_head_cpu: Optional[str] = None
        pathways_head_mem: Optional[str] = None
```